src/imgtools/modules/structureset.py

```
@@ -1,222 +1,450 @@
+		mask_points = physical_points_to_idxs(
+			reference_image, physical_points, continuous=continuous
+		)
+		for contour in mask_points:
+			try:
+				z, slice_points = np.unique(contour[:, 0]), contour[:, 1:]
+				if (
+					len(z) == 1
+				):  # assert len(z) == 1, f"This contour ({name}) spreads across more than 1 slice."
+					slice_mask = polygon2mask(size[1:], slice_points)
+					mask[z[0], :, :, idx] += slice_mask
+			except:  # rounding errors for points on the boundary
```

Ruff (0.8.2)

Do not use bare `except`

(E722)